2021-11-22 01:24:39,055 INFO    MainThread:127574 [wandb_setup.py:_flush():71] setting env: {'project': '11_21train'}
2021-11-22 01:24:39,055 INFO    MainThread:127574 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-22 01:24:39,056 INFO    MainThread:127574 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211122_012439-5n4g6l6j/logs/debug.log
2021-11-22 01:24:39,056 INFO    MainThread:127574 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211122_012439-5n4g6l6j/logs/debug-internal.log
2021-11-22 01:24:39,056 INFO    MainThread:127574 [wandb_init.py:init():404] calling init triggers
2021-11-22 01:24:39,056 INFO    MainThread:127574 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-22 01:24:39,057 INFO    MainThread:127574 [wandb_init.py:init():449] starting backend
2021-11-22 01:24:39,057 INFO    MainThread:127574 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-22 01:24:39,076 INFO    MainThread:127574 [backend.py:ensure_launched():199] starting backend process...
2021-11-22 01:24:39,087 INFO    MainThread:127574 [backend.py:ensure_launched():204] started backend process with pid: 127649
2021-11-22 01:24:39,088 INFO    MainThread:127574 [wandb_init.py:init():458] backend started and connected
2021-11-22 01:24:39,090 INFO    MainThread:127574 [wandb_init.py:init():519] updated telemetry
2021-11-22 01:24:39,112 INFO    MainThread:127574 [wandb_init.py:init():549] communicating current version
2021-11-22 01:24:40,243 INFO    MainThread:127574 [wandb_init.py:init():554] got version response 
2021-11-22 01:24:40,243 INFO    MainThread:127574 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-22 01:24:40,760 INFO    MainThread:127574 [wandb_init.py:init():592] starting run threads in backend
2021-11-22 01:24:41,577 INFO    MainThread:127574 [wandb_run.py:_console_start():1816] atexit reg
2021-11-22 01:24:41,578 INFO    MainThread:127574 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-22 01:24:41,579 INFO    MainThread:127574 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-22 01:24:41,581 INFO    MainThread:127574 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-22 01:24:41,581 INFO    MainThread:127574 [wandb_init.py:init():619] run started, returning control to user process
2021-11-22 01:24:41,590 INFO    MainThread:127574 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': 'firsttest_network=resnet,layers=32', 'no': '0', 'model': 'resnet', 'print_freq': 10, 'layers': 32, 'droprate': 0.3, 'cardinality': 8, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': True, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00040_network=resnet,layers=32', 'commit': 'not_set'}
2021-11-22 01:25:09,066 INFO    MainThread:127574 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-22 01:25:09,067 INFO    MainThread:127574 [wandb_run.py:_restore():1758] restore
2021-11-22 01:25:11,147 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:25:11,746 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:25:11,850 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9886
}

2021-11-22 01:25:11,953 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9886
}

2021-11-22 01:25:12,057 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9886
}

2021-11-22 01:25:12,160 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9886
}

2021-11-22 01:25:12,264 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9886
}

2021-11-22 01:25:12,367 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9886
}

2021-11-22 01:25:12,470 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:12,573 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:12,676 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:12,784 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:12,886 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:12,990 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:13,378 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}

2021-11-22 01:25:14,431 INFO    MainThread:127574 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9886
  total_bytes: 9886
}
local_info {
}

2021-11-22 01:25:15,529 INFO    MainThread:127574 [wandb_run.py:_append_files():2184] logging synced files
